src/Xamarin.Android.Build.Tasks/Tasks/ResolveLibraryProjectImports.cs

```diff
@@ -422,6 +424,16 @@ public override bool RunTask ()
 			}
 		}
 
+		void CreateResourceArchive (string resDir, string outputFile)
```

Comment:
If we go with this, I think I'd want to know the build-time impace of creating `res.zip` then if `aapt2` command are any slower using the `.zip` instead of a directory that is already extracted.

Reply:
This is interesting. 

### Old

```
2118 ms  ResolveLibraryProjectImports               1 calls
```

### Res.Zip

```
2387 ms  ResolveLibraryProjectImports               1 calls
```

So it is slightly slower to generate the `res.zip` file. This is on an AndroidX based project so its generating about 15 of those files. However this is not the full story. Lets take a look at what happens with the `aapt2 compile` calls.

### Old

```
581 ms  Aapt2Compile                               1 calls
```

### Res.Zip

```
366 ms  Aapt2Compile                               1 calls
```

It seems that not traversing the directory actually speeds up the `aapt2 compile` slightly 🤷